package.json

```diff
@@ -7,7 +7,7 @@
     "build": "remix vite:build",
     "dev": "remix vite:dev",
     "start": "remix-serve ./build/server/index.js",
-    "lint": "biome check --apply",
+    "lint": "biome check --apply .app/",
```

Comment:
it's typo :p
Also no need to specifiy the path here, since it has now specified in the cofig file: https://github.com/VoiceDeck/app/blob/main/biome.json#L3-L4